src/Symfony/Component/ErrorHandler/ErrorRenderer/TxtErrorRenderer.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\ErrorRenderer;
+
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+
+/**
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class TxtErrorRenderer implements ErrorRendererInterface
+{
+    private $debug;
+    private $charset;
+
+    public function __construct(bool $debug = true, string $charset = null)
+    {
+        $this->debug = $debug;
+        $this->charset = $charset ?: (ini_get('default_charset') ?: 'UTF-8');
```

Comment:
@yceruto , the `$charset` property is not used in this class. Consider removing it?

Reply:
Removing it in https://github.com/symfony/symfony/pull/31398/files#diff-ae80b93077d647cfffa8d40755febdb1 Thanks !